cloud_governance/main/es_uploader.py

```
@@ -95,14 +94,13 @@ def __get_user_cost(self, data):
                 num += 1
         return user_cost_results
 
-    def upload_last_policy_to_elasticsearch(self, policy: str, index: str, doc_type: str, s3_json_file: str,
+    def upload_last_policy_to_elasticsearch(self, policy: str, index: str, s3_json_file: str,
                                             es_add_items: dict = None):
```

Ruff (0.15.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)